client/app/account/component/AccountInfo.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { MdAccountCircle } from 'react-icons/md'
+import { colorMap } from '@/_consts/iconColor'
+import { UserType } from '@/_types'
+
+type AccountInfoProps = {
+  user: UserType
+}
+
+const AccountInfo = ({ user }: AccountInfoProps) => {
+  return (
+    <div className='flex flex-col gap-6 font-semibold'>
+      <div className='flex flex-col'>
+        <div></div>
```

Comment:
Delete unnecessary div!

Reply:
fix it! 
8b448ff822baa353a3e0b55f56b2897a2ba4f855